src/main/java/com/cglx/Calculator.java

```diff
@@ -6,6 +6,7 @@
 import java.io.FileReader;
 import java.util.ArrayList;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.util.ArrayList'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-01:9990/coding_rules#rule_key=squid%3AUselessImportCheck)